var/spack/repos/builtin/packages/rocm-smi-lib/package.py

```diff
@@ -116,7 +116,11 @@ class RocmSmiLib(CMakePackage):
     patch("disable_pdf_generation_with_doxygen_and_latex.patch", when="@4.5.2:")
 
     def cmake_args(self):
-        return [self.define_from_variant("BUILD_SHARED_LIBS", "shared")]
+        args = [
+            self.define_from_variant("BUILD_SHARED_LIBS", "shared"),
+            self.define("CMAKE_INSTALL_LIBDIR", "lib"),
```

Comment:
Do you want `"lib"` or `self.prefix.lib`?

Reply:
@tldahlgren , Update as recommended above.
Thank you..

Reply:
You do want `"lib"`, see [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html). But to be honest: why do you want this? The reason you end up with `lib64` is because you use `GNUInstallDirs` in that project. So it's expected that the libs to into `lib64` on CentOS.

Reply:
Hi @haampie, 
This change was indented to match with the RPM packaging distros of ROCm releases.
Library path as per the RMP distro is `/opt/rocm-<ver>/rocm_smi/lib/` in CentOS.
I added this change to align with this.